src/runner/index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8" />
     <script src="https://unpkg.com/decimal.js@10.3.1/decimal.js"></script>
     <script src="https://unpkg.com/big.js@6.1.1/big.js"></script>
-    <script src="./patches.js"></script>
+    <script type="module" src="./patches.js"></script>
```

Comment:
Now we can import constants and not worry about keeping them in sync. 🎉

Reply:
I thought for sure I tried this and it didn't find the `../constants.js` import, I assumed it was because webpack bundled it somehow, but maybe I just made a dumb typo.